src/style.css

```diff
@@ -1 +1,69 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+}
+
+.stars--1 {
+  display: flex;
+}
+
+.stars--2 {
+  display: flex;
+}
+
+.stars--3 {
+  display: flex;
+}
+
+.stars--4 {
+  display: flex;
+}
+
+.stars--5 {
+  display: flex;
+}
+
+.darkstars {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.goldstars {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
do not set here styles as already be settled for .stars__star.
here should be settled only background-image. use :nth-child pseudo class and stars--1 ... stars--5 modifiers for it